.vscode/settings.json

```diff
@@ -1,13 +1,13 @@
 {
-	"[typescript]": {
-		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
-		"editor.formatOnSave": true
-	},
-	"[typescriptreact]": {
-		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
-		"editor.formatOnSave": true
-	},
-	"eslint.run": "onType",
-	"eslint.format.enable": true,
-	"typescript.tsdk": "node_modules/typescript/lib"
-}
\ No newline at end of file
+  "[typescript]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+    "editor.formatOnSave": true
+  },
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+    "editor.formatOnSave": true
+  },
+  "eslint.run": "onType",
+  "eslint.format.enable": true,
+  "typescript.tsdk": "node_modules/typescript/lib"
+}
```

Comment:
Where did the tabs go?

Reply:
Looks like ESLint ignored this file and didn't add the tabs automatically...